example/app.go

```diff
@@ -124,3 +124,14 @@ func bugFun1() {
 	fmt.Println("Value of num is: ", num)
 	fmt.Println("Value of val is: ", val)
 }
+
+func bugFun2() {
+	var num int = 5
+	var ptr *int
+	var val intafdsa
+	ptr = &num
+	val = *ptr + 2
+	fmt.Println("Value of num is: ", num)
+	fmt.Println("Value of val is: ", val)
+	9 = a + b //fdsafd
+}
```

Comment:
by looking at the code patch. This patch adds a new function called bugFun2 which has some errors in it. The first error is that the variable val is declared as intafdsa, which is not a valid type. The second error is that there is an expression 9 = a + b which is not valid syntax. 

To fix these errors, we can replace intafdsa with int and delete the expression 9 = a + b.